packages/deck-scripts/config/basePluginRollupConfig.js

```diff
@@ -0,0 +1,55 @@
+const baseRollupConfig = require('./baseRollupConfig');
```

Comment:
This looks like it was moved from the pluginsdk.  Did git detect that it was a "rename"?  

I was wondering if git had a mechanism to retain history like this.  It turns out it doesn't.  I'd really like to create one commit where this file gets moved from the pluginsdk and a second commit where the pluginsdk config is re-created with the `module.exports = require('......` so we can retain history on this file.